src/System.Globalization/tests/CultureNotFoundException/CultureNotFoundExceptionTests.cs

```diff
@@ -11,43 +9,60 @@ namespace System.Globalization.Tests
     public class CultureNotFoundExceptionTests
     {
         [Fact]
-        public void TestMessage()
+        public void Ctor_String()
         {
-            CultureNotFoundException cnf = new CultureNotFoundException("this is a test string");
-            Assert.Equal("this is a test string", cnf.Message);
+            string message = "this is a test string";
+            CultureNotFoundException cultureNotFoundException = new CultureNotFoundException(message);
+
+            Assert.Equal(message, cultureNotFoundException.Message);
         }
 
         [Fact]
-        public void TestInnerExceptionMessage()
+        public void Ctor_String_Exception()
         {
-            CultureNotFoundException cnf = new CultureNotFoundException("this is a test string", new Exception("inner exception string"));
+            string message = "this is a test string";
+            Exception innerException = new Exception("inner exception string");
+            CultureNotFoundException cultureNotFoundException = new CultureNotFoundException(message, innerException);
 
-            Assert.Equal("this is a test string", cnf.Message);
-            Assert.Equal("inner exception string", cnf.InnerException.Message);
+            Assert.Equal(message, cultureNotFoundException.Message);
+            Assert.Same(innerException, cultureNotFoundException.InnerException);
         }
 
         [Fact]
-        public void TestParamName()
+        public void Ctor_String_String()
         {
-            CultureNotFoundException cnf = new CultureNotFoundException("aNameOfAParam", "this is a test string");
-            Assert.Equal("this is a test string" + System.Environment.NewLine +"Parameter name: aNameOfAParam", cnf.Message);
+            string paramName = "nameOfParam";
+            string message = "this is a test string";
+            CultureNotFoundException cultureNotFoundException = new CultureNotFoundException(paramName, message);
+
+            Assert.Equal(paramName, cultureNotFoundException.ParamName);
+            Assert.Equal(message + Environment.NewLine + "Parameter name: " + paramName, cultureNotFoundException.Message);
```

Comment:
I'm not sure - do you suggest I change it to something like `Assert.NotNull(cultureNotFoundException.Message)`